README.md

```diff
@@ -72,6 +77,26 @@ For more info check `Examples/arcs-make help`.
 
 To run the `arcs` executable in default mode, run `arcs <alignments>`. For descriptions of all arguments, run `arcs --help`.
 
+### Running ARCS in '--arcs-long' mode
+
+The arcs-long mode uses alignments of long reads to scaffold the input contigs.
```

Comment:
```suggestion
The arcs-long mode first simulates pseudo-linked reads from long reads, then uses alignments of the pseudo-linked reads to scaffold the input contigs.
```
Or something similar? Thinking just so they don't think we align the full long reads